editor/modes/visual-editor/block.js

```diff
@@ -364,6 +376,19 @@ class VisualEditorBlock extends Component {
 						<div className="editor-visual-editor__block-controls">
 							<BlockSwitcher uid={ block.uid } />
 							<Slot name="Formatting.Toolbar" />
+							<Toolbar className="editor-visual-editor__mobile-tools">
+								{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
+								{ ( showUI || isHovered ) && <BlockRightMenu uid={ block.uid } /> }
+								<IconButton
+									className="editor-visual-editor__mobile-toggle"
+									onClick={ this.toggleMobileControls }
+									aria-label={ __( 'Toggle extra block controls' ) }
```

Comment:
Redundant with `label` prop below (`IconButton` uses `label` to apply `aria-label`).